crates/gitbutler-stack/tests/mod.rs

```diff
@@ -1,5 +1,5 @@
-pub mod file_ownership;
-pub mod ownership;
```

Comment:
Is there a tool we can use to detect unused pubs within a workspace?

Reply:
Not that I know of. For integration tests these are never required to be `pub`, so it's an easy fix.